plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -48,6 +48,7 @@ export const ImagesSection: React.FC = () => {
 				( file ) => file.id === parseInt( image?.props?.id, 10 )
 			);
 		} );
+		recordEvent( 'product_image_change_order' );
```

Comment:
Actually, can we track specifically when the user actually clicks the change image order arrow buttons? If so...

Suggestion: `product_images_change_image_order_button_click`

If not, then `product_images_change_image_order_via_image_gallery`

Reply:
After taking a glance at the package, it seems like it's not possible to track only when the button is clicked. We have an `onOrderChange` event that is triggered after clicking the button or after dropping an image to change the order. 
I renamed the event in the commit [806a256](https://github.com/woocommerce/woocommerce/pull/35262/commits/806a25636222dabbd36210b6dafe71413dc6cfbe)